modules/node_modules/@ncigdc/containers/FileTable.js

```diff
@@ -94,13 +100,18 @@ export const FileTableComponent = (props: TTableProps) => {
                   padding: '3px',
                 }}
               >
-                <AddToCartButtonAll
-                  edges={props.hits.edges.map(e => e.node)}
-                  total={props.hits.total}
-                />
+                {!props.isCartPage &&
```

Comment:
referring back to my other comment.
https://github.com/NCI-GDC/portal-ui/pull/1273#discussion_r103765514